### ISSUE TITLE
SqueezePlayer is always started

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#102]
```
What steps will reproduce the problem?
1. Install SqueezePlayer app
2. Disable automatic SqueezePlayer start on Squeezer start
3. Exit Squeezer and SqueezePlayer
4. Start Squeezer again

What is the expected output? What do you see instead?
SqueezePlayer should not be started, but it always gets started

What version of the product are you using? On what operating system?
Squeezer 1.1.0-beta-2, Nexus 7 with Android 4.4.4

```

Original issue reported on code.google.com by `thenktor` on 2014-08-21 15:17:03

**Comment by kaaholst** [comment]
```
You're right this will be fixed before the new version is released.

Note that the author of SqueezePlayer also made some changes in his app to integrate
better with Squeezer, which he will release when we release our new version. This means
that you will get an even better integration, when the official (i.e. non-beta) versions
come out.
```

Original issue reported on code.google.com by `kaaholst` on 2014-08-21 20:26:20

**Comment by kaaholst** [comment]
```
Fixed before 1.1.0 was released.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2014-11-04 15:13:44
